README.md

```diff
@@ -14,18 +14,63 @@ All bugs, stories, and backlog for this project are managed through [Hyperledger
 in project IS (note that regular Indy tickets are in the INDY project instead...). Also, join
 us on [Hyperledger's Rocket.Chat](https://chat.hyperledger.org/) at #indy-sdk to discuss.
 
-## Building Indy SDK
+## How-to-install
+### Instal for Ubuntu based distro (Ubuntu 16.04)
+It is recommended to install packages with APT (change stable to `master` or `rc` if needed):
+    
+    apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 68DB5E88
+    sudo add-apt-repository "deb https://repo.sovrin.org/sdk/deb xenial stable"
+    sudo apt-get update
+    sudo apt-get install -y libindy
+    
+### Instal for Windows
+1. follow to https://repo.sovrin.org/windows/libindy.
+2. download last version of libindy and indy-sdk-deps.
```

Comment:
Please re-check this point: just for install libindy sdk-deps should not be required. As it should be fully included into libindy distribution package

Reply:
Also it would be nice to provide exact name (or pattern) of artifact

Reply:
"Instal" --> "Install" (2 Ls)